src/screens/Onboarding/Layout.tsx

```diff
@@ -35,7 +36,7 @@ export function Layout({children}: React.PropsWithChildren<{}>) {
   const {gtMobile} = useBreakpoints()
   const onboardDispatch = useOnboardingDispatch()
   const {state, dispatch} = React.useContext(Context)
-  const scrollview = React.useRef<ScrollView>(null)
+  const scrollview = React.useRef<Animated.ScrollView>(null)
```

Comment:
I have no idea why it won't let me use `ScrollView` here. Oh well, not a huge deal